include/zephyr/sys_clock.h

```diff
@@ -289,6 +289,25 @@ static inline bool sys_timepoint_expired(k_timepoint_t timepoint)
 	return K_TIMEOUT_EQ(sys_timepoint_timeout(timepoint), Z_TIMEOUT_NO_WAIT);
 }
 
+/**
+ * @brief Compare two timepoint values.
+ *
+ * This function is used to compare two timepoint values and find the one that is pointing
+ * to earlier point in time. It does not verify if given timepoints are already expired.
+ *
+ * @param a Timepoint to compare
+ * @param b Timepoint to compare against.
+ * @return zero if both timepoints are the same. Negative value if timepoint A points to earlier
+ * time, positive otherwise.
```

Comment:
Nit suggested brief:

```suggestion
 * @brief Compare two timepoint values.
 *
 * This function is used to compare two timepoint values.
 *
 * @param a Timepoint to compare
 * @param b Timepoint to compare against.
 * @return zero if both timepoints are the same. Negative value if timepoint @a a is before
 * timepoint @a b, positive otherwise.
```